platform/search/src/services/GenericSearchProvider.js

```diff
@@ -223,7 +226,7 @@ define([
             .then(function (objects) {
                 delete provider.pendingIndex[idToIndex];
                 if (objects[idToIndex]) {
-                    provider.index(idToIndex, objects[idToIndex].model);
+                    provider.index(idToIndex, objects[idToIndex].model, objects[idToIndex].model.type);
```

Comment:
Alittle confused about this, if we are already passing in the model, why we do also need to pass in model.type? Wouldn't the type property be available from model?

Reply:
That's a very good point. Let's chat about that...